src/main/java/com/leverx/springapp/demo/DemoApplication.java

```diff
@@ -1,27 +1,13 @@
 package com.leverx.springapp.demo;
 
-import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.context.annotation.Configuration;
 
 @SpringBootApplication
-@RestController
+@Configuration
```

Comment:
@blackavgn why we need it?

Reply:
Why we need RestController?I dont have it in my DemoApplication.java